internal/config/docs.toml

```diff
@@ -0,0 +1,624 @@
+# TODO check changelog for notes on undocumented fields
+
+# Setting to `true` enables development mode. This setting is not recommended for production deployments. It can be useful for enabling experimental features and collecting debug information.
```

Comment:
This is brilliant, I might suggest for a bit of additional clarity we add the field name as prefix (a bit like godoc forces you to) e.g.

```
# Dev: Setting to `true` enables development mode...
```

Or potentially, with newline:

```
# Dev:
# Setting to `true` enables development mode...
```

Reply:
I thought about that, but it is already duplicated in the markown heading and code block, so a triple stutter might be overkill?
![Screenshot from 2022-06-07 12-09-12](https://user-images.githubusercontent.com/1194128/172442109-fa1fe55c-8a6e-4e72-b48b-51eb943c6cdf.png)

Reply:
Not possible to add it to the comment in generated example TOML but leaving it out of the generated markdown docs?

Reply:
Maybe. I have just been focusing on generating the markdown format, and only using this toml as pure data input for the markdown, and not as an example on it's own :thinking:

Reply:
Begining the description with the short field name is enforced by the test now. We can tweak the presentation if we want, but I'm fine with it :shrug: 
![Screenshot from 2022-06-09 07-19-17](https://user-images.githubusercontent.com/1194128/172845543-a09a69db-ab72-410d-9bba-15e1107d0466.png)
One easy option would be to insert the table name so the description begins with the full key:

## Database.Backup
...
### Mode
```toml
Mode = 'none' # Default
```
Database.Backup.Mode sets the type of...